.config/nextest.toml

```diff
@@ -1,7 +1,10 @@
 [profile.default]
-retries = { backoff = "exponential", count = 10, delay = "1s", jitter = true, max-delay = "10s" }
 status-level = "all"
 final-status-level = "slow"
 failure-output = "immediate-final"
 fail-fast = false
 slow-timeout = { period = "30s", terminate-after = 4 }
+retries = { backoff = "exponential", count = 10, delay = "250ms", jitter = true, max-delay = "10s" }
+
+[profile.ci]
+retries = { backoff = "exponential", count = 10, delay = "250ms", jitter = true, max-delay = "10s" }
```

Comment:
Do we need this separate profile, or could we just reduce the `retries.delay` value in the default profile?